examples/built-in-integer-subtypes/built_in_integer_subtypes.out

```diff
@@ -0,0 +1,6 @@
+bal run built_in_integer_subtypes.bal
+72
+43543
+error: {ballerina}NumberConversionError {"message":"'int' value '4,294,967,296' cannot be converted to 'lang.int:Unsigned32'"}
```

Comment:
If user run this code, he will get a runtime error. Is it ok?
Isn't it better if we catch the error in the Ballerina Code and print `'int' value '4,294,967,296' cannot be converted to 'lang.int:Unsigned32'`.

Reply:
It's okay IMO. Panics are part of the language. Trapping it will bring in extra (unrelated) concepts to the BBE.